modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/index/meta/IndexMeta.java

```diff
@@ -27,6 +27,37 @@
  * Index meta information.
  */
 public class IndexMeta extends IndexMetaKey {
+    /** Index type. */
+    public enum IndexType {
```

Comment:
Maybe separate class file?

Reply:
This class is only a couple of lines long, what's the point? Also, `IndexType` is a very broad name, it's nice to have it nested in `IndexMeta` to better understand, where this type is used.

Reply:
Ok